npm-shrinkwrap.json

```diff
@@ -4877,14 +4877,14 @@
       "resolved": "https://registry.npmjs.org/moment-duration-format/-/moment-duration-format-1.3.0.tgz"
     },
     "mountutils": {
-      "version": "1.0.6",
-      "from": "mountutils@1.0.6",
-      "resolved": "https://registry.npmjs.org/mountutils/-/mountutils-1.0.6.tgz",
+      "version": "1.1.0",
+      "from": "mountutils@latest",
```

Comment:
@lurch Looks like if you do `npm install --save dependency@x.y.z`, then the shrinkwrap file includes that `x.y.z` version in the `from` property. If you simply do `npm install --save dependency`, the last version is implicit, and the shrinkwrap file will contains "latest"